dev.env

```
@@ -5,7 +5,7 @@
 #
 # Image: ckan
 CKAN_SITE_ID=default
-CKAN_SITE_URL=http://dev-adr
+CKAN_SITE_URL=dev-adr:5000
```

Maybe we could set the our local domain in our dev.env file? By default let it be dev-adr, but it's defined here and everywhere else we try to use a variable instead of hardcoded values??
```suggestion
CKAN_SITE_FQDN=dev-adr
CKAN_SITE_CONTAINER_PORT=5000
CKAN_SITE_URL=${CKAN_SITE_FQDN}:${CKAN_SITE_CONTAINER_PORT}
```

---

It's just a stop gap really. After we merge nginx into adx_develop it'll be easier
`CKAN_SITE_URL=http://dev-adr`

Also ckan requires the protocole `http` to be part of the URL. I've pushed a fix to this to the PR.